thundra/wrappers/aws_lambda/lambda_application_info_provider.py

```diff
@@ -1,9 +1,22 @@
+import uuid
+
 from thundra import constants, utils
 from thundra.application.application_info_provider import ApplicationInfoProvider
 
 
 class LambdaApplicationInfoProvider(ApplicationInfoProvider):
-    application_info = {}
+
+    def __init__(self):
+        log_stream_name = utils.get_env_variable(constants.AWS_LAMBDA_LOG_STREAM_NAME)
+        function_version = utils.get_env_variable(constants.AWS_LAMBDA_FUNCTION_VERSION)
+        function_name = utils.get_env_variable(constants.AWS_LAMBDA_FUNCTION_NAME)
+
+        self.application_info = {
+            'applicationId': '',
+            'applicationInstanceId': log_stream_name.split(']')[1] if log_stream_name else str(uuid.uuid4()),
```

Comment:
Might this `log_stream_name.split(']')[1]` fail if log stream name is not in valid format (no `]` exist for ex)